### DISCUSSION TITLE
Firebase message Subscribe to topic is not implemented

### DISCUSSION THREAD

**Discussion by kaundinya5** [firebase/flutterfire#4721] [General]
I'm using firebase_messaging ^8.0.0-dev.14 and I was trying to use the `subscribeToTopic` and I got a `Dart Unhandled Exception: MissingPluginException` So I went into the library to see what the function is doing and this is what I found

`Future<void> subscribeToTopic(String topic) {
    throw UnimplementedError('subscribeToTopic() is not implemented');
  }`

Is there any Idea when this will be implemented? I'm not able to use firebase_messaging 7.0.3 since I'm also using firebase_auth 0.20.0+1 and that requires firebase_core 0.7.0

**Comment by Salakar** [comment]
Was this on web?

**Reply by kaundinya5** [reply]
No, this was on Android.
